docs/reference/indices/seal.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[indices-seal]]
+== Seal
+
+The seal API allows to flush one or more indices and adds a marker to primaries and replicas if there are no pending write operations.
+The seal marker is used during recovery after a node restarts. If a replica is allocated on a node which already has a shard copy with the same seal as the primary then no files will be copied during recovery.
+Sealing is a best effort operation. If write operations are ongoing while the sealing is in progress then writing the seal might fail on some copies.
+
```

Comment:
I was trying to comment on it but it was easier to just rewrite some of it and make assumptions then to ask questions. So I sent #11235. Is it right?